messages.yml

```diff
@@ -0,0 +1,89 @@
+# commands:
+helloilaris_help: "Greets the Ilaris Bot."
+helloilaris_reply: "Hey, <@{author}>!"
+
+creatures_help: "Gets link to KreaturenDB"
+creatures_desc: "Name of creature"
+creatures_url: "https://ilaris-online.de/app/kreaturen/{params}"
+
+ilaris_help: "Posts an image of the given page if argument is numeric. Else the database entry."
+ilaris_desc: "Number or database entry (for example 4 or 'Duplicatus')"
+
+card_help: "Posts an image of a specified rule card."
+card_desc: "Name of rule card"
+card_best_matches: "Die drei besten Matches sind: {best}"
+
+r_help: |
+  Rolls dice.
+  2d6+3: Sum the result of 2 6-sided dice and 3.
+  2@3d20: Roll 3d20 and take the second highest i.e. (20,15,5) => 15.
+  Special rolls:
+  I: 1d20, Io: 1@2d20, Ioo 1@3d20
+  III: 2@3d20, III': 2@4d20, III'' 2@5d20
+r_desc: "Dice string to parse."
+r_result: |
+  ```md
+  # {author} --- {identifier} {result}
+  Details: {details}
+  ```
+
+group_prefix: "Your group (without username)"
+
+gcreate_help: "Creates a new group with yourself as the GM."
+gcreate_desc_default: "Eine spannende Ilaris Runde!"
+gcreate_desc: "Description for your soon-to-be players"
+gcreate_group: "Your group to create"
+gcreate_time: "Time to play in GMT+1 (for example '24.02.24 14:00')"
+gcreate_maxplayers: "Maximum number of players"
+gcreate_channel_created: "Hey, <@{author}>! Ich habe dir und deiner Gruppe diesen Kanal erstellt."
+gcreate_group_exists: "You already have a group called {name}..."
+
+glist_help: "Lists all joinable groups."
+glist_desc: "'True' to show full groups, too"
+
+gdestroy_help: "Destroys a group that you've created."
+group_deleted_info: "{author} hat die Gruppe {name} gelöscht. Du bist daher kein Mitglied mehr."
+
+gpurge_help: "Purges a group."
+gpurge_group: "Group (full name)"
+
+gset_help: "Sets a group key."
+gset_key: "Key to set. For example: {date},{description} or {players}"
+gset_value: "Value to set the key to"
+
+gsetdate_help: "Sets date."
+gsetdate_value: "New date"
+
+gsetnumberofplayers_help: "Sets maximum_players number."
+gsetnumberofplayers_value: "New number of players"
+
+gsetdescription_help: "Sets description."
+gsetdescription_group: "Your group (without username)"
+gsetdescription_value: "New description"
+
+gremove_help: "Removes a player from your group"
+gremove_player: "Player to remove."
+gremove_info: "Verabschiedet <@{player}>."
+
+gjoin_help: "Join a group as a player"
+gjoin_group: "Group you will join."
+gjoin_info: "Begrüßt <@{player}>!"
+
+gleave_help: "Leave a group as a maximum_players"
```

Comment:
this should be player. VVas already vvrong vvhen I vvrote it in the first place. Probably a replacement error.